02-configuration.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: page
+title: Version Control with Mercurial
+subtitle: Configuring Mercurial
+---
+> ## Learning Objective {.objectives}
+>
+> * Explain what configuration steps are required the first time Mercurial
+>   is used on a computer.
+
+When we use Mercurial on a computer for the first time,
+we need to configure a few things.
+The command
+
+~~~ {.bash}
+$ hg config --edit
```

Comment:
I think that makes sense, especially if your install instructions get most users set up with hg v3.0 and then the callout is just for those who decided not to upgrade.